docs/src/modules/components/ApiPage.js

```diff
@@ -298,6 +298,14 @@ function ApiDocs(props) {
         <Typography variant="h5" component="p" className="description" gutterBottom>
           {description}
         </Typography>
+        <Heading hash="demos" />
+        <p
+          dangerouslySetInnerHTML={{
+            __html:
+              'To see demos and examples on how to use this React component, visit the component guide section:',
```

Comment:
Here's how I would phrase this:

```suggestion
              'For examples and details on the usage of this React component, visit the component demo page:',
```

Reply:
Another thought—to teach users where to find the demos in the side nav:

```suggestion
              'For examples and details on the usage of this React component, visit the corresponding <b>Component</b> page:',
```

Reply:
The second proposal doesn't work with MUI X and makes the line wrap. I have used the first one.